src/LiveComponent/templates/deferred.html.twig

```diff
@@ -0,0 +1,7 @@
+<{{ loadingTag }} {{ attributes }} data-action="live:connect->live#$render">
+    {% block loadingContent %}
```

Comment:
Would this working ? 

```
<twig:Notifications defer>
    <p>Loading...</p>
</twig:Notifications>

```

If not, i thing this could be a great DX addition... wdyt ?  

If you'd want to use the "default" content block you'd just have to name it 

```
<twig:Notifications defer>
   {# Displayed on page load #}
    <p>Loading...</p>

    {# Displayed on demand #}
    <twig:block name="content">
       No notification
    </twig:block>
</twig:Notifications>
```

Reply:
I've checked this trying to do
```twig
<twig:SomeLiveComponent lazy>
    {% block loadingContent %}
        Loading...
    {% endblock %}
</twig:SomeLiveComponent>
```

But it renders `SomeLiveComponent` instantly : (. I'll try to debug how it works, maybe some adjustments and it'll work.